docs/navigation-configuration.md

```diff
@@ -1,22 +1,236 @@
+
+
+### Path parameters
+
+Use the path parameter values to define the **pathSegment** in your configuration. You can use a static value for your **pathSegment**, but you can add a colon to this value (for example `:projectId`) and make it act as a parameter. This tells Luigi to accept any value for this **pathSegment** of the main application URL. The parameter is replaced by the value when it is further processed by the application.
+
+A sample structure with a parametrized **pathSegment** is as follows:
+```
+{
+  navigation: {
+    nodes: [
+      {
+        pathSegment: 'home',
+        label: 'Home',
+        viewUrl: 'https://my.microfrontend.com/',
+        children: [
+          {
+            pathSegment: 'projects',
+            label: 'Projects',
+            viewUrl: 'https://my.microfrontend.com/projects/list.html',
+            children: [
+              {
+                pathSegment: ':projectId',
+                label: 'Project Details',
+                viewUrl: 'https://my.microfrontend.com/projects/details.html#id=:projectId;'
+              }            
+			]
+          }
+        ]
+      }
+    ]
+  }
+}
+
+```
+
+ Use the following options to work with path parameters:
+
+- Add the parameters to the **viewUrl** by placing them anywhere in the **viewUrl** value. For example if the main application URL is `https://luigiexample.com/home/projects/pr23`, then the **viewUrl** of the micro front-end in the content area would be `https://my.microfrontend.com/projects/details.html#id=pr23`. 
+- Use the [Luigi Client API](luigi-client-api.md) to access the node parameter values from the micro front-end. Use the `LuigiClient.getPathParams()` function. 
+For example, to get the value of the sample project parameter, use `LuigiClient.getPathParams().projectId`. 
+- Add a parameter to the context part of your configuration:
+  ```
+  {
+    pathSegment: ':projectId',
+    label: 'Project Details',
+    viewUrl: 'https://my.microfrontend.com/projects/details.html#id=:projectId;',
+    context: {
+    	project: ':projectId'
+    }
+  } 
+  ```
+In all cases the parameter is automatically replaced by the real value.
+
+
+### Node parameters
+
+Node parameters can be used to build the **viewUrl** and passed to the micro front-end specified in the navigation node selected in the navigation path. 
+You can specify them in the main application URL, similarly to URL query parameters with a specific prefix. The prefix is `~` by default, but you can reconfigure it using the global **nodeParamPrefix** setting. 
+
+All parameters without the prefix are not passed to the micro front-end and are consumed by the main application. 
+
+A sample **viewUrl** `https://luigiexample.com/home/projects/pr23?~sorting=asc&~page=2` supports sorting and paging (with the page number as a numerical value) by introducing the **sort** and **page** node parameters.
```

Comment:
```suggestion
A sample **viewUrl** `https://luigiexample.com/home/projects/pr23?~sorting=asc&~page=2` supports sorting and paging, with the page number as a numerical value, by introducing the **sort** and **page** node parameters.
```

Reply:
(with the page number as a numerical value) -> isn't it obvious? I'd consider removing this info.